ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/CreateViewBasedReportModal.tsx

```diff
@@ -84,18 +80,48 @@ function CreateViewBasedReportModal({
         >
             <Flex gap={{ default: 'gapMd' }}>
                 <FlexItem>
-                    Export a view-based CSV report from this view using the filters you&apos;ve
-                    applied. Once completed, this report will be available in the one time reports
-                    queue until it is purged according to your retention settings.
+                    Generate a CSV report based on this view and the filters you&apos;ve applied.
+                    Once completed, this report will be available in the view-based reports section
+                    until it is purged according to your retention settings.
                 </FlexItem>
                 {message?.type === 'success' && (
                     <Alert
                         variant="success"
                         isInline
-                        title={message.value}
+                        title="Report generation started successfully"
                         className="pf-v5-u-w-100"
                         component="p"
-                    />
+                    >
+                        {message.reportID && (
+                            <Flex
+                                direction={{ default: 'column' }}
+                                spaceItems={{ default: 'spaceItemsXs' }}
+                            >
+                                <FlexItem>
+                                    <strong>Report Name:</strong>{' '}
+                                    {message.requestName || message.reportID}
+                                </FlexItem>
```

Comment:
Observation, not change request. This seems okay for one data item. For two, recommend description list.